plugins/jira_workflow.py

```diff
@@ -34,7 +34,7 @@ def start_issue_workflow(github_issues, jira, assignee=None):
 def start_create_pull_requests_workflow(github_issues, jira, assignee=None):
     # Create PR task if those are not created
     for issue in github_issues:
-        pr_text = str('pr#' + issue.project + '#' + issue.id)
+        pr_text = str('pr_create#' + issue.project + '#' + issue.id)
```

Comment:
it is a good place to us `format`

```python
pr_text = "pr_create#{issue.project}#{issue.id}".format(issue=issue)
```

Not a blocker, but format is more readable and already converts to `str`

Reply:
OK